src/services/user.service.ts

```diff
@@ -1,30 +1,45 @@
-import { getRepository } from 'typeorm'
+import { getRepository, IsNull } from 'typeorm'
 
-import { User as UserType } from 'devu-shared-modules'
+import UserModel from '../model/users.model'
 
-import User from '../model/users.model'
+import { User, DeveloperAuth } from 'devu-shared-modules'
 
-const connect = () => getRepository(User)
+const connect = () => getRepository(UserModel)
 
-export async function create(user: UserType) {
+export async function create(user: User) {
   return await connect().save(user)
 }
 
-export async function update(user: UserType) {
-  const { id, email, schoolId, preferredName } = user
-  return await connect().update(id, { email, schoolId, preferredName })
+export async function update(user: User) {
+  const { id, email, externalId, preferredName } = user
+
+  if (!id) throw new Error('Missing Id')
```

Comment:
Why is update the only function to have additional validation? Is this superfluous or should we be adding additional validation for objects prior to DB calls?